svg/elements/feComposite.json

```diff
@@ -52,40 +52,52 @@
           "__compat": {
             "support": {
               "chrome": {
-                "version_added": "1"
+                "version_added": "1",
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
               },
               "chrome_android": {
-                "version_added": true
+                "version_added": true,
+                "notes": "BackgroundImage/BackgroundAlpha are not supported (see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
               },
               "edge": {
-                "version_added": "≤18"
+                "version_added": "≤18",
+                "notes": "BackgroundImage/BackgroundAlpha are not supported from version 79(see <a href='https://crbug.com/137230'>Chromium bug 137230</a>)."
```

Comment:
FYI version 79 is when edge moved to Blink, and lost the ability to support BackgroundImage.